### DISCUSSION TITLE
Searching multiple points

### DISCUSSION THREAD

**Discussion by zxdawn** [earthaccess-dev/earthaccess#490] [Q&A] [answered]
I see CMR supports searching multiple points with `or` option: https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#c-point. How to use earthaccess for that? Thanks!

**Comment by mfisher87** [comment, marked as answer]
We currently support only a single point filter (https://earthaccess.readthedocs.io/en/latest/user-reference/granules/granules-query/#earthaccess.search.DataGranules.point), but this would be great to add. Opened #491
